03-lection1/01-button/button.scss

```
@@ -0,0 +1,51 @@
+button {
+	margin: 0;
+	padding: 0;
+	border: none;
+	background: none;
+}
+
+.options {
+	margin: 10px 5px;
+	padding: 10px 0;
+	button {
+		width: 180px;
+		height: 48px;
+		border-radius: 8px;
+		color: white;
+		font-size: 16px;
+    line-height: 19px;
+    transition: opacity 200ms ease-in-out, all 200ms ease-in-out;
+
+    &.primary {
```

```suggestion
    .button {
        &_primary {
        }
        
        &_secondary {
        }
    }
```

В данном случае модификаторы можно сделать так)